shopyo/shopyoapi/cmd.py

```diff
@@ -64,6 +65,10 @@ def initialise():
     SEP_CHAR = "#"
     SEP_NUM = 23
 
+    print("Cleaning Db before Initialization")
+    print(SEP_CHAR * SEP_NUM, end="\n\n")
+    clean(app)
+
```

Comment:
@Yahyakiani it might be better to call clean in manage.py. See https://github.com/Abdur-rahmaanJ/shopyo/pull/336/commits/c2cb14dd5e836f4e0ebdd034c208b9ef6bf9b0e4. That way we separate cleaning and initialization and also make writing tests easier.